### DISCUSSION TITLE
multi tuning feature

### DISCUSSION THREAD

**Discussion by sugizo** [CoderLine/alphaTab#462] [Ideas]
# Description
pls add multi tuning feature
**backgroud use case** 
in rolling stones songs : "Honky Tonk Women", "Brown Sugar" and "Start Me Up". Keith Richard use fifth strings guitar with open G Tuning (G-D-G-B-D) as rhythm, while Ronnie Wood use standard tuning
**ref**
https://en.wikipedia.org/wiki/Open_G_tuning#Overtones_of_the_fundamental_note_G

# Possible Solutions
```
\track "Guitar" "gtr."
\staff{tabs} \tuning d4 b3 g3 d3 g2
(0.1 0.2 0.3 0.4 0.5).1 {ch "G"} |

\track "Guitar" "gtr."
\staff{tabs} \tuning e4 b3 g3 d3 a2 e2
:4 3.6 0.5 2.5 0.4 2.4 |
```

**expected result** 
the tuning show in above staff or track for each of it, open g tuning on the top and standard tuning on bottom staff

**tested on**
https://sugizo.pythonanywhere.com/music/notation/form_tablature_in_notation

# Current Workarounds
alphatab just show one tuning, expected result alphatab show 2 tunings

thanks

**Comment by Danielku15** [comment]
Sounds like a good feature. Guitar Pro shows tunings in case of multi-track display just side by side. alphaTab currently just takes the first one. 
![image](https://user-images.githubusercontent.com/674916/101795739-7abe4880-3b08-11eb-8132-7f5c3122a21b.png)

Added https://github.com/CoderLine/alphaTab/issues/463 into v1.2 but maybe I will also shift it to the backlog.

**Reply by sugizo** [reply]
please be awared about chord definition too, because it will have different intepretaion
e.g.
```
\track "Keith Gtr"
\staff{tabs} \tuning d4 b3 g3 d3 g2 \chord "G" 0 0 0 0 0
(0.1 0.2 0.3 0.4 0.5).1 {ch "G"} |

\track "Ronnie Gtr"
\staff{tabs} \tuning e4 b3 g3 d3 a2 e2 \chord "G" 3 0 0 0 2 3
(3.1 0.2 0.3 0.4 2.5 3.6).1 {ch "G"} |
```

**expected result**
the chord defined show below tuning definition for each of it
if not defined around the tunning, the open strings sound of standard tuning will produce Chord **Em11** not **G Mayor** Chord (expected result)

**tested on**
https://sugizo.pythonanywhere.com/music/notation/form_tablature_in_notation

thanks

**Reply by Danielku15** [reply]
Rendering of the tuning was "beautified" a bit and we render now tunings of all tracks displayed. The track name is shown above the tuning in case multiple tunings are displayed: 
![image](https://user-images.githubusercontent.com/674916/103486479-839bf480-4dfe-11eb-9da4-bf08df3a49ff.png)

Chords of all tracks were already displayed, but there was an issue with the unique chord ids in alphaTex.
